nft_ingester/src/api/dapi/search_assets.rs

```diff
@@ -148,7 +148,14 @@ async fn fetch_assets<
     let keys = index_client
         .get_asset_pubkeys_filtered(filter, &sort_by.into(), limit, page, before, after, &options)
         .await
-        .map_err(|e| StorageError::Common(e.to_string()))?;
+        .map_err(|e| {
+            if e.to_string().contains("statement timeout") {
+                StorageError::QueryTimedOut
+            } else {
+                StorageError::Common(e.to_string())
+            }
+        })?;
+
```

Comment:
would be cool if we could statically type the error in IndexDbError, but if it's not possible then i'm ok with what written here and below